secure/storage/src/tests/suite.rs

```diff
@@ -383,7 +383,7 @@ fn test_create_sign_rotate_sign(storage: &mut dyn Storage) {
 
 fn create_ed25519_key_for_testing() -> Ed25519PrivateKey {
     let mut rng = StdRng::from_seed([13u8; 32]);
-    Ed25519PrivateKey::generate_for_testing(&mut rng)
+    Ed25519PrivateKey::generate(&mut rng)
```

Comment:
Why we didn't use`generate_for_testing` here? Is there a specific reason to have this extra test rng?

Reply:
good question, I should have cleaned this up... historically there was no "modern" interface for generating a Ed25519PrivateKey hence you see this fragmentation.